<?xml version="1.0"?>
<launch>
	<node name="accelerationLimit" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="accelerationLimit"/>
	<node name="auxCtrl" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="auxCtrl"/>
	<node name="cartDampingHigh" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="cartDampingHigh"/>
	<node name="jointDampingHigh" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="jointDampingHigh"/>
	<node name="jointDampingLow" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="jointDampingLow"/>
	<node name="cartIntegratorGain" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="cartIntegratorGain"/>
	<node name="armCartJoint" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="armCartJoint"/>
	<node name="cartStiffnessHigh" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="cartStiffnessHigh"/>
	<node name="jointStiffnessHigh" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="jointStiffnessHigh"/>
	<node name="jointStiffnessLow" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="jointStiffnessLow"/>
	<node name="kp" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="kp"/>
	<node name="kTendon" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="kTendon"/>
	<node name="lowPriorityJointRef" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="lowPriorityJointRef"/>
	<node name="taskNodes" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="taskNodes"/>
	<node name="velocityLimit" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="velocityLimit"/>
	<node name="trajGenSwitch" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="trajGenSwitch"/>
	<node name="includeWaistJacobian" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="includeWaistJacobian"/>
	<node name="tendonBoost" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="tendonBoost"/>
	<node name="fingerImpedanceSwitch" pkg="r2_dynamic_reconfigure" type="fake_reconfigure_server.py" args="fingerImpedanceSwitch"/>
	<!--<node name="reconfigure_gui" pkg="dynamic_reconfigure" type="reconfigure_gui" respawn="true" />-->

</launch>

